src/algorithms/policy_gradient/ppo.jl

```diff
@@ -178,7 +178,12 @@ end
 function (agent::Agent{<:PPOPolicy})(env::MultiThreadEnv)
     dist = prob(agent.policy, env)
     action = rand.(agent.policy.rng, dist)
-    EnrichedAction(action; action_log_prob = logpdf.(dist, action))
+    if ndims(action) == 2
```

Comment:
Would maybe be nicer to dispatch on whether it was a "multi action env" or a "single action env" or something along those lines, then there wouldn't need to be this check each time.

Reply:
👍